### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by brannondorsey**
I'm proposing we adopt the Citizen Code of Conduct.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+ * Be mindful of your surroundings and of your fellow participants. Alert community leaders if you notice a dangerous situation, someone in distress, or violations of this Code of Conduct, even if they seem inconsequential.
+ * Remember that community event venues may be shared with members of the public; please be respectful to all patrons of these locations.
+
+## 4. Unacceptable Behavior
+
+The following behaviors are considered harassment and are unacceptable within our community:
+
+ * Violence, threats of violence or violent language directed against another person.
+ * Sexist, racist, homophobic, transphobic, ableist or otherwise discriminatory jokes and language.
+ * Posting or displaying sexually explicit or violent material.
```
Comment (brannondorsey) [inline CODE_OF_CONDUCT.md]: Maybe we should change this line slightly. I wholly agree that sexually explicit material can be damaging and offensive, but its also a subject that rightfully comes up in art communities, and I don't think we should outright ban sexually explicit material from our community. Rather, we should define what types of sexually explicit material is appropriate.

**Comment by brannondorsey** [comment]
Will close #47

**Comment by cvalenzuela** [comment]
Thanks for creating this @brannondorsey! I think now is a good time to start thinking about the code of conduct for not just this repo, but for the whole Runway community. -- Before we merge this, perhaps @shiffman(😃) might help us in terms of good practices and things we should have in mind when writing this doc.

**Comment by shiffman** [comment]
I'd love to help with this @cvalenzuela!

**Comment by cvalenzuela** [comment]
Yey! Great! Would love to know if you have any suggestions or ideas on how to structure this document so we can foster a healthy and open community from the very beginning. I know p5 has a very nice and [custom code of conduct](https://github.com/processing/p5.js/blob/master/CODE_OF_CONDUCT.md). What was the process of writing that doc?

**Comment by brannondorsey** [comment]
Amazing, thanks @shiffman! The CoC in this PR is the Citizen's Code of Conduct as championed by [Stumptown Syndicate](http://stumptownsyndicate.org/). It's one of the two templates GitHub provides if you create a new file via the web interface named `CODE_OF_CONDUCT.md` (similar to what happens if you create a `LICENSE` file). I read both proposed CoCs, and this one felt more community oriented, which is why I proposed it here, but the wording is definitely not something I am tied to.

**Comment by rubychilds** [comment]
Related to this, I have started an incident report form as stated - https://docs.google.com/forms/d/19iwryiOPCZ0JN2-8hsjxV-n0Fy5bNgXF2cRjl7ispSA/edit?usp=sharing. This is based off both the Docker incident report form and Recurse Center method for internal incidents. 

I will take a look at the CoC more indepth now + provide a fork of any recommended updates.

**Comment by brannondorsey** [comment]
Thanks for these improvements @rubychilds!

**Pull request merged**